### DISCUSSION TITLE
Benchmark/performance ?

### DISCUSSION THREAD

**Discussion by bloor** [pocketbase/pocketbase#417] [Q&A] [answered]
Can we get some direct performance comparison with MySQL/ProgrestSQL and some nonsql databases ?

- max multiple connections
- read/write operations 
- latency

I'm currently targeting a gaming-project that could reach 20k active users. But the database will get bigger and bigger when it will come to storing game-details and statistics.

**Comment by ganigeorgiev** [comment, marked as answer]
Hi! The project is still in its early development stage but once we reach a little more stable version I will crete a proper testbed to track the performance and eventual release regressions.

It is hard to tell if PocketBase will be suitable for your use case. SQLite can store terabytes of data. It depends on your hardware and mostly how many concurrent writes do you expect, because SQLite supports only 1 writer at a time. Writes are pretty fast and I've tried to keep the transactions short to not block other queries, but still it is something that you should have in mind when choosing a backend for your project. One quick tip to speed up the default PocketBase write operations is by compiling with CGO enabled (the writes are roughly ~2 times faster with the cgo driver).

You can also check the official SQLite "When to use" guide - https://www.sqlite.org/whentouse.html.

**Reply by bloor** [reply]
Aight.. thanks for the reply. 
Waiting for 1.0 👋 

**Personally I think the project it's insane value.** It's what I always envisioned back in my PHP days when using Yii2.. a easy to use/manipulate the database while also having a solid REST API.
